README.rst

```diff
@@ -1398,9 +1398,19 @@ Choose from the list of available rules:
     ``'newest'``
   - ``use_class_const`` (``bool``): use ::class notation; defaults to ``true``
 
-* **php_unit_ordered_covers** [@PhpCsFixer]
+* **php_unit_ordered_annotations** [@PhpCsFixer]
 
-  Order ``@covers`` annotation of PHPUnit tests.
+  Order annotations of PHPUnit tests.
+
+  Configuration options:
+
+  - ``annotations`` (a subset of ``['covers', 'uses']``): list of annotations to
+    order, e.g. ``["covers"]``; defaults to ``[]``
```

Comment:
Should we include `covers` by default, or should the default be an empty `array`?

Reply:
I think `covers` as default is nice for the people migrating to the new fixer